gradient/cli/common.py

```diff
@@ -242,3 +242,15 @@ def _compose_object_key(self):
 class GradientDatasetOption(GradientObjectListOption):
     def __init__(self, param_decls, **kwargs):
         super(GradientDatasetOption, self).__init__("dataset", param_decls, **kwargs)
+
+
+def validate_comma_split_option(comma_option_value, option_value):
+    if comma_option_value or option_value:
+        if option_value:
+            option_value = list(option_value)
+        else:
+            option_value = list()
+        if comma_option_value:
+            comma_option_value = [s.strip(" ") for s in comma_option_value.split(",")]
```

Comment:
You don't have to pass argument to .strip(). If you pass a string it won't strip other white characters so it's better to just execute without parameters here.